json-rpc/spec.json

```diff
@@ -196,6 +196,101 @@
 				"$ref": "#/components/contentDescriptors/GasPrice"
 			}
 		},
+		{
+			"name": "eth_feeHistory",
+			"summary": "Transaction fee history",
+			"description": "Returns transaction base fee and effective priority fee history for the requested/supported block range.",
+			"params": [
+				{
+					"name": "blockCount",
+					"description": "Number of blocks in the requested range.",
```

Comment:
Consider mentioning what a value of `0` does/means.  Also, are there any constraints on this (limit to number one can request)?

Reply:
done